src/tiled/newtilesetdialog.ui

```diff
@@ -275,9 +275,23 @@
         </item>
        </layout>
       </item>
-      <item row="0" column="1" colspan="3">
+      <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="image"/>
       </item>
+      <item row="1" column="3">
+       <widget class="QToolButton" name="dropperButton">
+        <property name="enabled">
+         <bool>false</bool>
+        </property>
+        <property name="text">
+         <string/>
+        </property>
+        <property name="icon">
+         <iconset resource="tiled.qrc">
+          <normaloff>:/images/22x22/stock-tool-by-color-select.png</normaloff>:/images/22x22/stock-tool-by-color-select.png</iconset>
```

Comment:
I've just pushed `:/images/16x16/stock-tool-color-picker-16.png`, please use that icon instead.